src/imu/sensors/imu_sensors/sensor_vector.cpp

```diff
@@ -0,0 +1,29 @@
+#include "sensor_vector.hpp"
+
+namespace imu {
+
+auto SensorVector::Update(void) noexcept -> types::DriverStatus {
+  SensorGeneral::GetRawValues();
+
+  if (ImuConnectionSuccessful()) {
+    SetSensorValues(
+        ConvertUint8BytesIntoInt16SensorValue(raw_values_.at(0), raw_values_.at(1)),
```

Comment:
Everything except -1, 0, 1 is a magic number. What is the justification to use magic numbers here?

Reply:
I couldn't get up with a better name. This is the order of bytes sent over I2C. I could just name them "Byte_0, Byte_1, Byte_2" and so on. But thats not really helpful. What do you think?

Reply:
Ah I see. That is not really evident from the implementation. You can either name the magic numbers (I agree with you, that is not really nice to look at) or just leave a short comment, that this is the byte order.